client/app/templates/application.hbs

```diff
@@ -2,11 +2,10 @@
 <div class="container-fluid">
   <div class="row">
     <div class="col-md-offset-2 col-sm-8 col-md-8 main">
-      {{add-game}}
+      {{add-game model=model.games saveGameInParent=(action 'saveGameInParent')}}
```

Comment:
When using the name onSave, this line could become:

model=model.games onSave=(action 'saveGame')